app/src/main/res/values/strings.xml

```diff
@@ -253,4 +253,23 @@
     <string name="sort_by_balance">Balance</string>
     <string name="reverse">Reverse</string>
     <string name="sort_by">Sort by</string>
+    <string name="version_history">Version History</string>
+    <string name="current_version_2_6_1">Current Version: 2.6.1</string>
+    <string name="arrow"><![CDATA[>]]></string>
+    <string name="credits">Credits</string>
+    <string name="copyright_u00a9_2019_2021_sylvia_van_os">Copyright © 2019-2021 Sylvia van Os.</string>
+    <string name="donate">Donate</string>
+    <string name="show_your_appreciation">Show your appreciation!</string>
+    <string name="help_translate_this_app">Help translate this app</string>
+    <string name="english_is_translated_100">English is translated 100%</string>
```

Comment:
Not really useful to have it always say 100%. Better to not show any info than anything else.